pkg/pipe/pipe.go

```diff
@@ -66,8 +66,7 @@ func (me MaskEngine) MaskContext(e model.Dictionary, key string, context ...mode
 		return copy, nil
 	}
 
-	for _, dict := range e.Get(key).([]model.Entry) {
-		elemInput := model.CleanTypes(dict).(model.Dictionary)
+	for _, elemInput := range model.CleanDictionarySlice(e.Get(key)) {
```

Comment:
It's possible here to iterate over untyped interfaces and use CleanDictionary method inside the loop. Doing this, the new method CleanDictionarySlice is not needed anymore

Reply:
With your propose change the test [TestMaskEngineShouldMaskNestedArray](https://github.com/CGI-FR/PIMO/pull/36/files#diff-ba00e8503a5538751e64dd8e698e7bd592c77719f729a48e6240a364afacd3e8R31) failed and we can't use `model.NewPipelineFromSlice`.